config/production.json

```diff
@@ -132,7 +132,7 @@
 		"woocommerce/extension-promotions": true,
 		"woocommerce/extension-reviews": true,
 		"woocommerce/extension-settings": true,
-		"woocommerce/extension-settings-email": true,
+		"woocommerce/extension-settings-email": false,
```

Comment:
Doesn't this still need to be set to true to pass [this check](https://github.com/Automattic/wp-calypso/blob/master/client/extensions/woocommerce/index.js#L216)